src/documents/activities/ToddlerGrandparentConversation.html.md.handlebars

```diff
@@ -0,0 +1,25 @@
+---
+title: The Toddler-Grandparent Conversation
+layout: content
+category: Activities
+---
+
+Toddlers repeatedly ask “why?”. Why is the sky blue? Why do cats have tails? In this activity, we participate in an imagined grandparent-toddler conversation by repeatedly asking “why?”. 
+
+This activity is a great way to unpack the motivations or the nested context of a design scenario. It demonstrates and exercises the third dimension of inclusive design – Broader Beneficial Impact – by taking us from the local and immediate assumptions out to their larger framing and context. 
```

Comment:
also it's not about going "in" from something. it's about starting from where we're at (the local and immediate), and going out to something bigger